app/controllers/dashboard_controller.rb

```diff
@@ -1,4 +1,6 @@
 class DashboardController < ApplicationController
+  before_action :require_user
```

Comment:
I wonder if we'll need a `BaseController` eventually that encapsulates the `before_action` for other controllers we've yet to make like the Viewing Party!